grapher/lineCharts/LineChart.tsx

```diff
@@ -622,7 +633,13 @@ export class LineChart
     }
 
     @computed get seriesStrategy(): SeriesStrategy {
-        return autoDetectSeriesStrategy(this.manager)
+        const hasNormalAndProjectedSeries =
+            some(this.yColumns, (col) => col.isProjection) &&
+            some(this.yColumns, (col) => !col.isProjection)
```

Comment:
You can safely use `Array#some` directly without using lodash, [a very tiny amount of browsers don't support it](https://caniuse.com/?search=Some).

(Related to that, we are most limited in the browsers we support by our use of Mobx 5, which requires support for [Proxy](https://caniuse.com/?search=Proxy).)